fe/fe-core/src/main/java/com/starrocks/sql/ast/UnitBoundary.java

```diff
@@ -0,0 +1,46 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+package com.starrocks.sql.ast;
+
+import com.starrocks.analysis.Expr;
+import com.starrocks.analysis.LiteralExpr;
+import com.starrocks.sql.common.ErrorType;
+import com.starrocks.sql.common.StarRocksPlannerException;
+import com.starrocks.thrift.TExprNode;
+
+public class UnitBoundary extends LiteralExpr {
```

Comment:
what's the meaning of UnitBoundary? why do we need to add this?

Reply:
We use it to specify START | END for time_slice, have add comments.